docs/Behaviour - Server Side.md

```diff
@@ -6,39 +6,29 @@ The initial state of Active Constraints of a Sender MUST be an empty `constraint
 
 Non-empty `constraint_sets` array MUST be treated as unordered unless at least one Constraint Set has `urn:x-nmos:cap:meta:preference` attribute.
 
-Once a Sender accepts proposed Active Constraints, this Sender, the Flow and the Connection API `/transportfile` resource (if used) associated with this Sender and the Source associated with this Flow MUST satisfy the Active Constraints when the Sender is active.
-
-### Request Methods
-
-- `GET` request returns the last successfully applied Active Constraints.
-- `PUT` request MUST be validated and if the Sender supports Parameter Constraints used in the proposed Active Constraints and is capable to adhere at least one of the Constraint Sets, then the associated Inputs may be reconfigured, which may eventually cause the update of the corresponding Flows, Sources, `/transportfile` and Sender. If successful, the operation MUST return the accepted Active Constraints, otherwise an error MUST be returned and nothing changed.
-- `DELETE` request MUST clear Active Constraints.
+Active Constraints are satisfied when any of its Constraint Sets are satisfied in terms of [BCP-004-01][BCP-004-01].
 
 ## State of Sender
 
 A Sender managed with IS-11 has the following states:
 - `Unconstrained` when Active Constraints of this Sender is an empty `constraint_sets` array.
-- `Constrained` when this Sender, the Flow and the `/transportfile` (if used) associated with this Sender and the Source associated with this Flow satisfy the Active Constraints.
-- `Active Constraints Violation` when this Sender, the Flow or the `/transportfile` (if used) associated with this Sender or the Source associated with this Flow does not satisfy the Active Constraints. When an inactive Sender is in this state, it MUST NOT allow [IS-05][IS-05] activations.
+- `Constrained` when the Active Constraints are satisfied.
```

Comment:
"when at least one of the Active Constraints is satisfied."

Reply:
Active Constraints is an object with a single property "constraint_sets". I added "Active Constraints are satisfied when any of its Constraint Sets are satisfied in terms of [BCP-004-01][BCP-004-01]" to define what Active Constraints satisfaction means. If any properties are added to Active Constraints, it will be possible to change this single line instead of editing all the places referring to the satisfaction.

Reply:
ok